src/AvaloniaMath/Rendering/AvaloniaElementRenderer.cs

```diff
@@ -1,103 +1,108 @@
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using Avalonia;
 using Avalonia.Media;
-using AvaloniaMath.Rendering;
+using AvaloniaMath.Fonts;
+using WpfMath;
 using WpfMath.Boxes;
+using WpfMath.Rendering;
 using WpfMath.Rendering.Transformations;
+using IBrush = WpfMath.Rendering.IBrush;
 
-namespace WpfMath.Rendering
+namespace AvaloniaMath.Rendering;
+
+/// <summary>The renderer that uses Avalonia drawing context.</summary>
+internal class AvaloniaElementRenderer : IElementRenderer
 {
-    /// <summary>The renderer that uses Avalonia drawing context.</summary>
-    internal class AvaloniaElementRenderer : IElementRenderer
+    private static readonly ISolidColorBrush DefaultForegroundBrush = Brushes.Black;
+
+    private readonly DrawingContext _foregroundContext;
+    private readonly double _scale;
+
+    public AvaloniaElementRenderer(DrawingContext foregroundContext, double scale)
     {
-        private readonly DrawingContext _foregroundContext;
-        private readonly double _scale;
+        _foregroundContext = foregroundContext;
+        _scale = scale;
+    }
 
-        public AvaloniaElementRenderer(DrawingContext foregroundContext, double scale)
-        {
-            _foregroundContext = foregroundContext;
-            _scale = scale;
-        }
+    public void RenderElement(Box box, double x, double y)
+    {
+        //   var guidelines = GenerateGuidelines(box, x, y);
```

Comment:
Are you still need these snippets?

Reply:
This will be the subject of further investigation in #325.